### PR TITLE
SAK-32667 Initial commit of Noodle's SiteDescription widget

### PR DESCRIPTION
**Pull request by adrianfish**

### CODE REVIEW

**Comment by ern** [comment]
empty file in the project

**Comment by adrianfish** [comment]
@ern Which file? empty.js?

**Comment by ern** [comment]
Yep!

**Comment by adrianfish** [comment]
that has to be there. It's wicket. Wicket insists on deploying a version of jquery and this is often a different version to the one we use. So, you have to force Wicket to use a particular version and set its default version to an empty file. Steve wrote this code - I'm just pushing it for Noodle.

**Comment by ern** [comment]
Could you then leave a comment to that effect in that file!

**Comment by adrianfish** [comment]
Yep, will do.

**Comment by steveswinsburg** [comment]
Yay this is finally going in. I know that NYU had a bunch of feedback on it though. @jeffpasch

**Comment by jeffpasch** [comment]
Sorry for the delay in responding.

Would it be possible to retain the ability for instrcutors to edit Site information display from the Home page by implementing a modal. e.g.,



This would save the user a good number of clicks (that we'd otherwise be introducing) and would how to edit this area more self-evident for users.

File: sitedescription/pom.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        <dependency>
+            <groupId>org.apache.wicket</groupId>
+            <artifactId>wicket-extensions</artifactId>
+            <version>6.18.0</version>
+        </dependency>
+        <!--dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+        </dependency-->
+        <dependency>
```
Comment (ern) [inline sitedescription/pom.xml]: Slf4j is global in sakai so can remove.

File: sitedescription/pom.xml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+        <!--dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+        </dependency-->
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```
Comment (ern) [inline sitedescription/pom.xml]: lombok is Global so this is not needed

**Comment by mgolden-noodle** [comment]
@jeffpasch Our view of this wasn't to put a giant block of text describing the course at the top. In fact, it was really just to put the title of the course with something like "Click 'Sessions' on the left to get started". No one needs to see a big course description thingy more than once, so it doesn't need to be above the Commons.

**Comment by adrianfish** [comment]
@ern I've removed the slf4j and lombok stuff, and added a comment to the empty js file.

**Comment by adrianfish** [comment]
@ern I'm waiting for your Godlike approval of this :)

**Pull request merged**